src/disclosure-manager/_disclosureCallbacks.js

```diff
@@ -0,0 +1,26 @@
+let disclosureCallbacks = [];
```

Comment:
@tbiethman should this get pushed under private API?

Reply:
I waffled on this. orion-application consumes it and imports it directly. I'm not sure if we want orion-application to be importing private APIs. By making it private, we'd be giving ourselves free-reign to make non-passive changes to it here and maybe breaking orion-application in the process.

I think it makes sense to keep it public (to keep us honest with passivity) but to hide it as much as humanly possible (i.e. no consumer facing docs in dev site).